config.toml

```diff
@@ -44,15 +44,19 @@ logoHomeLink = "https://ggilmore.github.io/tep-alumni-website"
 
 [[languages.en.menu.main]]
 identifier = "about"
-name = "About (National)"
+name = "TEP National"
 url = "/about"
 
 [[languages.en.menu.main]]
-identifier = "giving-alumxi"
-name = "Donate to AlumXi"
-url = "/giving/alumxi"
+identifier = "capital-campaign"
+name = "2022 Capital Campaign"
+url = "/giving/capital"
 
 [[languages.en.menu.main]]
-identifier = "giving-mit"
-name = "Donate to MIT"
-url = "/giving/mit"
+identifier = "giving-alumxi"
+name = "Donate!"
+url = "/giving/alumxi"
+#[[languages.en.menu.main]]
```

Comment:
@cjosephson could you remove this entirely if we're not using this page

Reply:
ideally it would have been done beforehand, but it's fine to make a new PR for this